tests/integration/session_endianness/test.rflx

```diff
@@ -25,7 +25,7 @@ package Test is
 
       state Copy is
       begin
-         Out_Msg := Messages::Msg_LE'(C => In_Msg.A, D => In_Msg.B);
+         Out_Msg := Messages::Msg_LE'(C => In_Msg.X_A, D => In_Msg.X_B);
```

Comment:
The input in the corresponding config should be adapted to the new message type. Currently, an incomplete `In_Msg` is used. That doesn't lead to an error, as `In_Msg.Y` is not accessed here, but I don't think we want to allow using incomplete messages in the long term (#935).

Reply:
Yes that was an oversight which the test didn't detect. Fixed.